### ISSUE TITLE
Client brukva connection

### ISSUE THREAD

**Issue by burakdede** [imvalgo/brukva#20]
```
class MessageUpdatesHandler(BaseHandler):
@tornado.web.authenticated
@tornado.web.asynchronous
def post(self):
    self.listing_id = self.get_argument("listing_id", None)
    self.cursor = self.get_argument("cursor", None)
    self.client = brukva.Client()
    self.client.connect()
    self.client.subscribe(self.listing_id)
    self.client.listen(self.on_new_messages)

def on_new_messages(self, messages):
    # Closed client connection
    if self.request.connection.stream.closed():
        return
    msg = json.loads(messages.body,'ascii')
    self.finish(dict(messages=[msg]))
    self.client.unsubscribe(self.listing_id)

def on_connection_close(self):
    # unsubscribe user from channel
    self.client.unsubscribe(self.listing_id)
    self.client.disconnect()
```

When I long poll using above code and check redis activity seems like when ever client send or posts something it creates new connection, so total connected client increasing on top of each other like 2 client talking and send 10 messages total connected client show 15 or sth like that. Is this bug or should I use different approach while connecting ?

**Comment by goldalworming** [comment]
what's your command in redis to monitor? I want to test it

**Comment by burakdede** [comment]
I am not using any special command just checking the redis server output and as I see there are increasing number of client connected in evert post

**Comment by burakdede** [comment]
@goldalworming did you find anything related to this problem ?

**Comment by goldalworming** [comment]
how do you paste your code in this commented system? 

I couldn't paste it...

**Comment by goldalworming** [comment]
just see my repo https://github.com/goldalworming/tornado-redis-chat

I use 'monitor' in redis-cli and it works fine (only 2 subscribers)
but I got weird error. maybe you could solve it.

**Comment by burakdede** [comment]
@goldalworming Yeah I do the same thing before but even its working it firing exception about "reading from non-existing connection" and try to listen channel its weird. And its probably related to this issue https://github.com/evilkost/brukva/issues/10
